ocs_ci/ocs/workload.py

```diff
@@ -47,23 +47,23 @@ def __init__(
 
         self.thread_exec = concurrent.futures.ThreadPoolExecutor(max_workers=1)
 
-    def setup(self, **setup_conf):
-        """
-        Perform work_load_mod.setup() to setup the workload.
-        Every workload module should implement setup() method so that
-        respective <workload_module>.setup() function can be called from here
-
-        Args:
-            setup_conf (dict): Work load setup configuration, varies from
-                workload to workload. Refer constants.TEMPLATE_WORKLOAD_DIR
-                for various available workloads
-
-        Returns:
-            bool: True if setup is success else False
-        """
-        if self.pod:
-            setup_conf["pod"] = self.pod
-        return self.work_load_mod.setup(**setup_conf)
+    # def setup(self, **setup_conf):
```

Comment:
Won't this affect the non-nginx pods ?

Reply:
@shylesh What do you think? Do we use Debian only for NGINX.. correct?

Reply:
@OdedViner I see another reference https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pod.py#L541 which is jenkins workload